docs/Support.md

```diff
@@ -0,0 +1,28 @@
+# Support Rotation
+
+Envoy Mobile has a support rotation system to ensure that discussions,
+issues and pull requests are handled in a timely manner from someone who
+works on Envoy Mobile at Lyft.
```

Comment:
```suggestion
issues and pull requests are handled in a timely manner from an Envoy Mobile maintainer.
```

Just because I'm an optimist: I think we can avoid using Lyft grouping so when our current friends are well onboarded and can help participate without being part of Lyft

Reply:
Some of the friends of Envoy Mobile have explicitly requested a rotation like this so they can reach someone within Lyft.

I'd love for there to be a more general Envoy Mobile rotation, and remove the need in having feedback from a specific company, at some point in the near future, at which point we can and should generalize the wording here.